packages/server/src/automations/steps/createRow.ts

```diff
@@ -15,7 +18,7 @@ export async function run({
 }: {
   inputs: CreateRowStepInputs
   appId: string
-  emitter: EventEmitter
+  emitter: ContextEmitter
```

Comment:
All of the automation action handlers wanted an `EventEmitter` as an argument. Digging down into why, I found that they actually _don't_ want an `EventEmitter` at all. This `emitter` is passed down to eventually go into a function called `buildCtx`, which creates a fake `UserCtx` object for use by action handlers to directly call controller endpoint functions. `UserCtx.eventEmitter` is of type `ContextEmitter`, so I've updated these types to match that.